src/assets/styles/base.scss

```diff
@@ -165,6 +165,7 @@ p {
 // Error Border
 .invalid-section {
   border-left: 3px solid $app-red;
+  margin-left: -3px;
```

Comment:
When we show a left border then we need to pull the element left by the same amount to maintain alignment.

Reply:
Does this work? and for the shared components as well?

I did try this originally but found some oddities. Maybe it was because things were already out of alignment before some most recent UX clean up. 

Either way, awesome if it does 👍

Reply:
I tested a couple of components only.

It might break components that already have left margin. Hmm.

Reply:
yeah, potentially an issue.  maybe just keep an eye on it for now and see if we see any weird alignment issues.

Reply:
There was no simple solution here so I decided to remove this. We can deal with it in the future if needed.